obci/scenarios/budzik_prototype/budzik_prototype.ini

```diff
@@ -0,0 +1,21 @@
+[peers]
```

Comment:
Let's move scenarios to obci/scenarious/budzik/prototypes/. 

@develancer: Can you update this pull request with the new paths?